exporters/logging-otlp/src/main/java/io/opentelemetry/exporter/logging/otlp/internal/traces/OtlpStdoutSpanExporterComponentProvider.java

```diff
@@ -30,7 +30,7 @@ public String getName() {
   }
 
   @Override
-  public SpanExporter create(DeclarativeConfigProperties config) {
+  public SpanExporter create(DeclarativeConfigProperties config, ComponentProviderLoader componentProviderLoader) {
```

Comment:
I think the vision was that you would be able to use `DeclarativeConfigProperties.getComponentLoader()`?